apache/superset
# fix(chart): legacyRegularTime and charts are being rendered, and solves issue #26181

### SUMMARY
This PR solves the issue mentioned in #26181. I have solved the logical error in `legacyRegularTime` and also made some changes in `superset-frontend/plugins/legacy-plugin-chart-calendar/src/controlPanel.ts` for better app structure.

### BEFORE
[![Before Changes]](https://github.com/apache/superset/assets/95441117/9a4ef0fa-69c0-4a47-a449-14e1907d99b0)

### AFTER
[![After Changes]](https://github.com/apache/superset/assets/95441117/b118e7e9-2eb6-4dd8-a3bb-9f81842b1f04)


### TESTING INSTRUCTIONS
Kindly create any chart which renders `legacyRegularTime` (Eg: Event Flow chart which is mentioned in #26181) and now you can be able to see desired result.

### ADDITIONAL INFORMATION
- [x] Has associated issue: Fixes #26181
